<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Petri Solver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>490</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionExportar_Diagrama"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionAgregar_Nodo"/>
    <addaction name="actionAgregar_Transicion"/>
    <addaction name="separator"/>
    <addaction name="actionGenerar_Modelo"/>
    <addaction name="actionCrear_Arcos"/>
    <addaction name="separator"/>
    <addaction name="actionEjecutar"/>
    <addaction name="actionBorrar"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionMatriz_Pre"/>
    <addaction name="actionMatriz_Post"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuVer"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAgregar_Nodo"/>
   <addaction name="actionAgregar_Transicion"/>
   <addaction name="separator"/>
   <addaction name="actionGenerar_Modelo"/>
   <addaction name="actionCrear_Arcos"/>
   <addaction name="separator"/>
   <addaction name="actionEjecutar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/fileopen.png</normaloff>:/Images/Resources/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/filesaveas.png</normaloff>:/Images/Resources/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionExportar_Diagrama">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/file_export.png</normaloff>:/Images/Resources/file_export.png</iconset>
   </property>
   <property name="text">
    <string>Exportar Diagrama</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/window_close.png</normaloff>:/Images/Resources/window_close.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAgregar_Nodo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/node.png</normaloff>:/Images/Resources/node.png</iconset>
   </property>
   <property name="text">
    <string>Agregar Nodo</string>
   </property>
  </action>
  <action name="actionAgregar_Transicion">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/transition.png</normaloff>:/Images/Resources/transition.png</iconset>
   </property>
   <property name="text">
    <string>Agregar Transicion</string>
   </property>
  </action>
  <action name="actionCrear_Arcos">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/arc.png</normaloff>:/Images/Resources/arc.png</iconset>
   </property>
   <property name="text">
    <string>Crear Arcos</string>
   </property>
  </action>
  <action name="actionEjecutar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/1rightarrow.png</normaloff>:/Images/Resources/1rightarrow.png</iconset>
   </property>
   <property name="text">
    <string>Ejecutar</string>
   </property>
  </action>
  <action name="actionGenerar_Modelo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/generate.png</normaloff>:/Images/Resources/generate.png</iconset>
   </property>
   <property name="text">
    <string>Generar Modelo</string>
   </property>
  </action>
  <action name="actionBorrar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/draw_eraser.png</normaloff>:/Images/Resources/draw_eraser.png</iconset>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </action>
  <action name="actionMatriz_Pre">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/matrix.png</normaloff>:/Images/Resources/matrix.png</iconset>
   </property>
   <property name="text">
    <string>Matriz Pre</string>
   </property>
  </action>
  <action name="actionMatriz_Post">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Resources/matrix.png</normaloff>:/Images/Resources/matrix.png</iconset>
   </property>
   <property name="text">
    <string>Matriz Post</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
